cpp/include/rmm/aligned.hpp

```diff
@@ -22,7 +22,7 @@
 #include <cstddef>
 #include <cstdint>
 
-namespace RMM_NAMESPACE {
+namespace RMM_EXPORT rmm {
```

Comment:
Is the idea to get rid of RMM_NAMESPACE altogether eventually?

Reply:
Yes, see: https://github.com/rapidsai/rmm/pull/1896#discussion_r2059528660